### PR TITLE
Fix bug related to click on outside handling and disabled class overflowing

### PR DESCRIPTION
**Pull request by Ghusar**
* Fix bug where null was being accessed while handle click on outside of dropdown
* Fix bug where disabled class was affecting other styles because of global declaration

### CODE REVIEW

File: src/main.css [hunk 1]
```diff
@@ -80,11 +80,11 @@
   border: 10px solid pink;
 }
 
-.disabled {
+.option.disabled {
```
Comment (ankit-m) [inline src/main.css:83]: Any specific reason why only options are disabled?

File: src/main.css [hunk 2]
```diff
@@ -80,11 +80,11 @@
...
   opacity: 0.5;
   background: #f0f0f0;
 }
 
-.disabled:hover {
+.option.disabled:hover {
   cursor: no-drop;
```
Comment (ankit-m) [inline src/main.css:89]: No drop?

File: src/main.css (hunk 1 above)
Comment (Ghusar) [reply src/main.css:83]: anything else we want disabled in the context of this dropdown?

File: src/main.css (hunk 2 above)
Comment (Ghusar) [reply src/main.css:89]: i think this must have been done to remove `cursor:pointer` while hovering on disabled option

File: src/main.css (hunk 1 above)
Comment (ankit-m) [reply src/main.css:83]: The trigger

**Review by ankit-m — APPROVED** [review]

**Pull request merged**